tests/html/Element/innerHTML.html

```diff
@@ -163,6 +172,26 @@
 
     document.body.removeChild(div);
   });
+
+  const SVG_NAMESPACE = 'http://www.w3.org/2000/svg';
+  const circleText = '<circle cx="50" cy="50" r="40" stroke="black" stroke-width="3" fill="red" />';
+
+  const canSVGInnerHTML = (function() {
+    // Edge 14 svg.innerHTML is broken even without the polyfill
+    const svg = NATIVE.createElementNS.call(document, SVG_NAMESPACE, 'svg');
+    NATIVE.innerHTML.call(svg, circleText);
+    return svg.querySelector('circle') instanceof SVGCircleElement;
+  })();
+
+  test('svg innerHTML', function() {
+    if (!canSVGInnerHTML) {
+      this.skip();
```

Comment:
Oh, this is nice to know about. I think there are a few other places in the tests where I ended up switching out the test function itself:
```
const canDoTheThing = ... some test ... ;
(canDoTheThing ? test : test.skip)('test name', function() { ...
```
Calling `this.skip()` in them would be much cleaner.